src/event-sources/aws/sqs.js

```diff
@@ -1,18 +1,24 @@
 const getRequestValuesFromSqs = ({ event }) => {
-  const method = 'POST'
```

Comment:
Could you revert the style changes in this file?

Reply:
I reverted these code